src/plot.jl

```diff
@@ -8,6 +8,7 @@
 
     xguide --> "Strategy"
     yguide --> "round"
+    label --> "score"
```

Comment:
I think you want this 
```suggestion
    colorbar_title --> "score"
```